### ISSUE TITLE
Fix localhost version of Mobile.GetMobileDocumentsPath

### ISSUE THREAD

**Issue by deadlyfingers** [thaliproject/Thali_CordovaPlugin#233]
Postcard app is now generating string error with Mobile.GetMobileDocumentsPath in 'mockmobile.js'

```
TypeError: Arguments to path.join must be strings
```

**Comment by vjrantal** [comment]
For cross-platform, you could do something like:
```
var os = require('os'); // This to the top of the file
return os.tmpdir();
```

**Comment by yaronyg** [comment]
This is supposed to be resolved by #63

**Comment by deadlyfingers** [comment]
@vjrantal - I submitted the update as a PR
